docs/dev/rfc_process.md

```diff
@@ -0,0 +1,103 @@
+# The Cirq RFC process
+
+New major Cirq feature begins life as a Request for Comment (RFC).
+
+An RFC is a document that describes a requirement and the proposed changes that will solve it. Specifically, the RFC will:
+
+*   Be formatted according to the [RFC template](https://tinyurl.com/cirq-rfc-template) in a Google Doc
+*   A github issue is submitted with the link to the Google Doc
+*   Be subject to discussion and a review meeting prior to acceptance.
+
+The purpose of a Cirq Request for Comments (RFC) is to engage the Cirq community in development, by getting feedback from stakeholders and experts, 
+and communicating design changes broadly.
+
+
+## What constitutes a major feature
+
+Major features can be one of the following:
+*   Features that drastically change the user experience or workflow of Cirq.
+*   Features that create new library dependencies for Cirq.
+*   New simulators, samplers, protocols, devices, or packages.
+*   Features that drastically change performance of Cirq.
+*   Features that maintainers disagree/debate about.
+
+The following are NOT major features:
+
+*   Fixing a bug.
+*   Extending the functionality of an existing method in a natural way.
+
+If you are not sure if a feature constitute as a “major feature”, just submit a Github issue with a description, 
+and one of the maintainers will flag the issue as a major feature if necessary.
+
+## How to submit an RFC
+
+
+1. Before submitting an RFC, discuss your aims with project contributors and maintainers and get early feedback. Maintainers can be contacted at cirq-maintainers@googlegroups.com
+2. Draft your RFC.
+    *   Follow the [RFC template](https://docs.google.com/document/d/1Zf0n7rKahejSJmAgYbCP6ClKXe8HhALceRQb-vRYe6o/edit#heading=h.mckr847qq60k).
+3. After writing the RFC draft, get feedback from maintainers and contributors before submitting it (email: cirq-maintainers@googlegroups.com) \
+Writing implementation code is not a requirement, but it may help design discussions.
+4. Recruit a sponsor.
+    *   A sponsor must be a maintainer of the project or the product manager
```

Comment:
It's a bit unclear how to get a sponsor.

Reply:
added more thoughts - PTAL.